### DISCUSSION TITLE
Cascadia Code PL - Which theme can be used out of the box?

### DISCUSSION THREAD

**Discussion by just1a-person** [JanDeDobbeleer/oh-my-posh#264] [Troubleshoot] [answered]
I am planning to rewrite the Powerline setup tutorial to use Oh-My-Posh v3 in the Microsoft Docs. I want to make it as easy as possible.

/cc @JanDeDobbeleer

**Comment by JanDeDobbeleer** [comment]
Good question. I adjusted most of the git icons to be compatible, so basically any theme that does not add too much nerd font only icons will do. My gut feeling fells me `agnoster` will do. Most out of the box themes do not rely heavily on icons.

**Reply by just1a-person** [reply]
Looks like `agnoster` will not do.

![Agnoster theme](https://user-images.githubusercontent.com/75578443/103152455-32f91b80-47ae-11eb-937e-9c9f0c7a1e1a.png)

**Reply by JanDeDobbeleer** [reply]
Can I ask why this is important? The use of Nerd Fonts is advised and was always added to the docs before. Maybe `star` has a better change of being friendly, but the chances of seeing incompatible icons is always a possibility.

**Reply by just1a-person** [reply]
Cascadia Code doesn't support a version with all the font icons yet. The Nerd Fonts have an older (patched) version of Cascadia Code. I want to make it as easy as possible with the latest version of Cascadia Code (PL) or any (latest) patched Cascadia Code.

**Reply by JanDeDobbeleer** [reply]
@itsme-alan maybe it's an idea to create Cascacia Code compatible theme in that case, I'd gladly accept PR's for that 👍🏻

**Comment by lnu** [comment]
There is a nerd font version of Cascadia, known as [Caskaydia Cove](https://github.com/ryanoasis/nerd-fonts/releases/download/v2.1.0/CascadiaCode.zip).(a bit outdated as you said, 6 months ago for the last commit).
![image](https://user-images.githubusercontent.com/1829553/104834965-95bc6f00-58a3-11eb-8369-a8764f7a8a2a.png)

And if you call `get-poshthemes`, you'll quickly get an idea of the working themes witch cascadia. Seems `zash` and `robbyrussel` are displayed correctly.

**Comment by JanDeDobbeleer** [comment, marked as answer]
As of yesterday, any theme with `.minimal` has out of the box support for Cascadia Code PL.
